src/Spectre.Algorithms/Methods/Segmentation.cs

```diff
@@ -66,7 +66,7 @@ public DivikResult Divik(IDataset dataset, DivikOptions options)
 
 			var varargin = options.ToVarargin();
 			var tmp = _segmentation.divik(numberOfOutputArgs, dataset.GetRawIntensities(), coordinates, varargin);
-			var result = new DivikResult(tmp);
+			var result = new DivikResult(tmp[1]);
```

Comment:
tmp[1] because tmp[0] is the same as field Partition in tmp[1], so we don't need tmp[0] at all